app/src/main/resources/eip-client.properties

```diff
@@ -22,3 +22,6 @@ camel.springboot.main-run-controller=true
 # Camel context name
 camel.springboot.name=${spring.application.name}
 # ----------------------------------------------------------------------------------------------------------------------
+
+# ---- Liquibase Configuration -----------------------------------------------------------------------------------------
+spring.liquibase.enabled=${ENABLE_LIQUIBASE}
```

Comment:
How are you utilizing this property in your code?

Reply:
It is required otherwise the `eip-demo` service fails to start
![Screenshot 2024-05-13 at 6 37 32 PM](https://github.com/ozone-his/eip-client/assets/36266597/2397e41a-600c-4bc0-a94e-2903c0ad48ce)